### ISSUE TITLE
streamline process_parameters

### ISSUE THREAD

**Issue by mikarubi** [mikarubi/voluseg#33]
Remove redundant validation of parameters from `process_parameters` (after #31 is merged) because this validation now occurs in `parameter_models`

**Comment by luiztauffer** [comment]
as of now, if a parameters file (json) already exists, the default behavior is to load from it.
should we have an optional flag, e.g. `overwrite: bool = False`, that allows the user to overwrite the local file with a new processed parameters dict?

**Comment by mikarubi** [comment]
No, the current setup is good. The idea is that once a parameter file is created, it should not be overwritten.
